src/schema/contracts/common/airdrop.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+import { AddressSchema, PriceSchema } from "../../shared";
+
+/**
+ * @internal
+ */
+export const AirdropAddressInput = z.object({
```

Comment:
check how i do it with snapshots if you wanna offer both string[] | AidropAddressInput[]